client/src/main/java/io/getstream/chat/android/client/errors/ChatError.kt

```diff
@@ -1,7 +1,8 @@
 package io.getstream.chat.android.client.errors
 
-open class ChatError : RuntimeException {
-    constructor(message: String?) : super(message)
-    constructor(message: String?, cause: Throwable?) : super(message, cause)
-    constructor(cause: Throwable?) : super(cause)
+open class ChatError(
+    val message: String?,
+    val throwable: Throwable? = null
```

Comment:
I don't have a strong preference either way, but we could consider calling this property something more concrete than its type, maybe `cause` as with the `Exception` classes.

Reply:
Good point! Thanks